.ci/furiosa-models-ci/lint.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: lint
+  namespace: ci-furiosa-models
+spec:
+  workspaces:
+    - name: source
+  params:
+    - name: image
+      description: The container image to use black in
+      default: asia-northeast3-docker.pkg.dev/next-gen-infra/furiosa-ai/furiosa-models:v0.8.0
```

Comment:
Just being curious, we don't need this specific tag right?

Reply:
I'll add another tag `lint` for that image and make use of it.